source/nodejs/adaptivecards-templating/typedoc.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "Adaptive Cards Templating Javascript SDK",
+	"inputFiles": "./src",
+	"mode": "library",
+	"out": "docs",
+	"exclude": [
+		"__tests__"
+	],
+	"excludeNotExported": true,
+	"excludePrivate": true,
+	"readme": "none",
```

Comment:
Why is this none and yet a ReadMe.md did get generated? What's the guidance here (i am hoping we'll eventually flip this switch to actually write up a readme that gets automatically included?

Reply:
[This option](http://typedoc.org/guides/options/#readme) allows you to either supply `none`, in which case it generates its own toplevel README.md linking directly into the docs, or you can supply `path/to/README.md`, which is the README it will use on the index page. IIRC, when I was configuring all this stuff for the `adaptivecards` package, not supplying any `readme` directive caused it to just use `source/nodejs/adaptivecards/README.md`. This meant that there was no aggregate page with links into all the various docs.